app/javascript/components/App.js

```diff
@@ -125,83 +130,104 @@ class App extends Component {
+        <main className="pt-24 min-h-screen">
+          <Switch>
+            <Route exact path="/" component={Home} />
+
+            {logged_in && (
+              <Route
+                path="/surfspotindex"
+                render={() => (
+                  <SurfSpotIndex
+                    surfSpots={this.state.surfSpots}
+                    collections={this.state.collections}
+                    createCollectionSpot={this.createCollectionSpot}
+                    super_secret_api_key={super_secret_api_key}
+                  />
+                )}
+              />
+            )}
 
-          {logged_in && (
             <Route
-              path="/surfspotindex"
-              render={() => <SurfSpotIndex 
-                surfSpots={this.state.surfSpots} 
-                collections={this.state.collections} 
-                createCollectionSpot={this.createCollectionSpot}
-                super_secret_api_key={super_secret_api_key}/>}
+              path="/surfspotshow/:id"
+              render={(props) => {
+                const id = props.match.params.id
+                const surfSpot = this.state.surfSpots.find(
+                  (spot) => spot.id === +id
+                )
+                return <SurfSpotShow surfSpot={surfSpot} />
+              }}
             />
-          )}
-
-          <Route
-            path="/surfspotshow/:id"
-            render={(props) => {
-              const id = props.match.params.id
-              const surfSpot = this.state.surfSpots.find(
-                (spot) => spot.id === +id
-              )
-              return <SurfSpotShow surfSpot={surfSpot} />
-            }}
-          />
-          <Route
-            path="/collectionnew"
-            render={() => (
-              <CollectionNew
-                user_id={current_user.id}
-                createCollection={this.createCollection}
+            <Route
+              path="/collectionnew"
+              render={() => (
+                <CollectionNew
+                  user_id={current_user.id}
+                  createCollection={this.createCollection}
+                />
+              )}
+            />
+            <Route
+              path="/collectionedit/:id"
+              render={(props) => {
+                const user_id = current_user.id
+                const id = props.match.params.id
+                const collection = this.state.collections.find(
+                  (collection) => collection.id === +id
+                )
+                return (
+                  <CollectionEdit
+                    collection={collection}
+                    editCollection={this.updateCollection}
+                    user_id={user_id}
+                  />
+                )
+              }}
+            />
+
+            <Route path="/aboutus" component={AboutUs} />
+
+            {/* using filter here to get the collections that bellongs to a specific user */}
+            {logged_in && (
```

Comment:
Unrelated to this pull request, but something to look into.  Does this need to be a separate conditional rendering from the other logged-in rendering?